packages/components/atoms/f-segmented-control/src/components/SegmentedControl.vue

```diff
@@ -0,0 +1,244 @@
+<template>
+    <div
+        :class="[
+            $style['c-segmentedControl'],
+            {
+                [$style['c-segmentedControl--large']]: size === 'large'
+            }
+        ]"
+        data-test-id="segmented-control"
+        :aria-label="screenreaderLabel"
+        role="radiogroup"
+        tabindex="-1"
+        @keydown="onKeyDown"
+    >
+        <button
+            v-for="(option, index) in options"
+            :key="option.label"
+            :data-test-id="`segmented-control-option-${index}`"
+            type="button"
+            :class="[
+                $style['c-segmentedControl-option'],
+                {
+                    [$style['c-segmentedControl-option--selected']]: option.label === selectedLabel
+                }
+            ]"
+            :aria-checked="option.label === selectedLabel ? 'true' : 'false'"
+            role="radio"
+            :tabindex="focusedTabIndex[index]"
+            :disabled="option.disabled"
+            @click="selectOption(option.label)"
+        >
+            <!-- TODO: This is a placeholder for adding vue icons -->
+            <span>{{ option.label }}</span>
+        </button>
+    </div>
+</template>
+
+<script>
+import { validateOptions } from './helpers/validators';
+
+export default {
+    name: 'SegmentedControl',
+    props: {
+        screenreaderLabel: {
```

Comment:
Do we need this as a separate prop? The user can just apply `aria-label`?

Reply:
I can see arguments for and against it. Using a prop means we can _require_ the consumer to provide a label which I quite like the idea of :) I can chat with Ash too when he's in!

Reply:
Just to cover what Jamie and myself just chatted about in person – if it's expected for users to add an aria label or description (so the component won't work accessibly without it), we should add props that are named like the above so that we can make them required or validate them.

@FayeCarter If that's the case for the toggle component (where we expect the aria labels/descriptions to be included), then might be worth adding those in as required props where they're named `screenreaderLabel` or something similar.